specs/CoreWebView2ControllerOptions.DefaultBackgroundColor.md

```diff
@@ -0,0 +1,133 @@
+CoreWebView2ControllerOptions.DefaultBackgroundColor
+===
+# Background
+
+Previously, there was a fix to address an issue where the background color controller property
+was applied too late, causing a disruptive white flash during the WebView2 loading process.
+
+This fix required using an environment variable. However, this workaround was not meant to be 
+a long-term solution. Therefore, we need to add this setting to `ICoreWebView2ControllerOptions` 
+to apply the color early in the loading process.
+
+# Description
+
+This interface extends the `ICoreWebView2ControllerOptions` to expose the `DefaultBackgroundColor` 
+property as an option.
+The `CoreWebView2ControllerOptions.DefaultBackgroundColor` API  allows users to set the 
+`DefaultBackgroundColor` property at initialization.
+
+This is useful when setting it using the existing [`CoreWebView2Controller.DefaultBackgroundColor API`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2controller.defaultbackgroundcolor?view=webview2-dotnet-1.0.2792.45)
+applies the color too late.
+
+
+
+# Examples
+
+## Win32 C++
+```cpp
+ HRESULT AppWindow::CreateControllerWithOptions()
+{
+    wil::com_ptr<ICoreWebView2ControllerOptions> options;
+    HRESULT hr = m_environment->CreateCoreWebView2ControllerOptions(&options);
+
+    if (hr == E_INVALIDARG)
+    {
+        ShowFailure(hr, L"Invalid profile name.");
```

Comment:
How can the default profile be invalid?

Reply:
From my understanding the  profile name must not be an empty string or a nullptr